CRM/Contact/Form/Search.php

```diff
@@ -464,7 +452,7 @@ public function buildQuickForm() {
           'class' => 'crm-form-submit',
         )
       );
-      $this->add('hidden', 'task', CRM_Contact_Task::GROUP_CONTACTS);
+      $this->add('hidden', 'task', CRM_Contact_Task::GROUP_ADD);
```

Comment:
I'm still reading but GROUP_ADD doesn't exist on my version that I can see

Reply:
It exists once #11240 is merged

Reply:
It is now in CRM/Core/Task.php which is part of core.